tests/unit/emr/test_connection.py

```diff
@@ -997,8 +1030,25 @@ def test_run_jobflow_service_role(self):
             'Action': 'RunJobFlow',
             'Version': '2009-03-31',
             'ServiceRole': 'EMR_DefaultRole',
-            'Name': 'EmrCluster' },
+            'Name': 'EmrCluster'},
             ignore_params_values=['ActionOnFailure', 'Instances.InstanceCount',
                                   'Instances.KeepJobFlowAliveWhenNoSteps',
                                   'Instances.MasterInstanceType',
                                   'Instances.SlaveInstanceType'])
+
+    def test_run_jobflow_enable_debugging(self):
+        self.region = 'ap-northeast-2'
+        self.set_http_response(200)
+        self.service_connection.run_jobflow(
+            'EmrCluster', enable_debugging=True)
+
+        actual_params = self.actual_request.params.copy().items()
+
+        expected_params = set([
+            ('Steps.member.1.HadoopJarStep.Jar',
+             's3://ap-northeast-2.elasticmapreduce/libs/script-runner/script-runner.jar'),
+            ('Steps.member.1.HadoopJarStep.Args.member.1',
+                's3://ap-northeast-2.elasticmapreduce/libs/state-pusher/0.1/fetch'),
+        ])
+
+        self.assertTrue(expected_params <= actual_params)
```

Comment:
I have to double check, but my reasoning was that assert_request_parameters test for list equality but I don't think this test should fail if the ordering of the params changes or if someone adds some new params to run_jobflow. 

Also, this isn't intended as length check, but a subset check (expected_params should be a subset of actual_params). I think I forgot to convert actual_params to a set, so I will fix this.